code/__DEFINES/mobs.dm

```diff
@@ -60,9 +60,17 @@
 #define DEFAULT_BODYPART_ICON_ORGANIC 'icons/mob/human_parts_greyscale.dmi'
 #define DEFAULT_BODYPART_ICON_ROBOTIC 'icons/mob/augmentation/augments.dmi'
 
+
 #define MONKEY_BODYPART "monkey"
-#define ALIEN_BODYPART "alien"
-#define LARVA_BODYPART "larva"
+#define ALIEN_BODYPART 	"alien"
+#define LARVA_BODYPART 	"larva"
+
+
+#define BODY_DEFAULT 0
+#define HUMAN_BODY 	(1 << 0)
```

Comment:
Don't use inline tabs.

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces

Reply:
fixed